### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Programmer026**
Screen Shot 2020-07-25 at 5 40 02 PM

Screen Shot 2020-07-25 at 6 32 41 PM

+ Changed font size for buttons
+ Added a box shadow to grid to show colored grid more clearly
+ Added stoke to text so it shows in both in light and dark colored grids

### CODE REVIEW

**Review by thuavu — APPROVED** [review]
Looks great!!!

File: src/main/webapp/css/style.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
     padding: 20px;
     text-align:center;
     text-decoration: none;
-    font-size: 50px;
+    font-size: 30px;
```
Comment (anamatasanch) [inline src/main/webapp/css/style.css]: This fix is good for now but it would be ideal for the text to resize based on the button size

**Review by anamatasanch — APPROVED** [review]

File: src/main/webapp/css/style.css [hunk 2]
```diff
@@ -10,7 +10,7 @@
     padding: 20px;
     text-align:center;
     text-decoration: none;
-    font-size: 50px;
+    font-size: 150%;
```
Comment (anamatasanch) [inline src/main/webapp/css/style.css]: I think we should increase the font size 😸

File: src/main/webapp/css/style.css [hunk 3]
```diff
@@ -65,6 +65,10 @@
 
 .gameButton {
     margin: none;
+    box-shadow: 0 0 1px 0
+        inset, 0 0 0 1px;
+    color: white;
+    -webkit-text-stroke: 0.3px black;
```
Comment (anamatasanch) [inline src/main/webapp/css/style.css]: I think the numbers inside the grid are too small and the stroke isn't very visible.

File: src/main/webapp/css/style.css [hunk 4]
```diff
@@ -65,6 +65,10 @@
 
 .gameButton {
     margin: none;
+    box-shadow: 0 0 1px 0
+        inset, 0 0 0 1px;
```
Comment (anamatasanch) [inline src/main/webapp/css/style.css]: I would rather this be a shadow that matches the color of the gameButton. I think the white border looks a little harsh.

**Review by anamatasanch — CHANGES_REQUESTED** [review]

File: src/main/webapp/css/style.css (hunk 4 above)
Comment (anamatasanch) [reply src/main/webapp/css/style.css]: Or something similar. I am not sure.

Comment (Programmer026) [reply src/main/webapp/css/style.css]: I was not too sure what to do, so I took out the shadow

File: src/main/webapp/css/style.css (hunk 3 above)
Comment (Programmer026) [reply src/main/webapp/css/style.css]: Screen Shot 2020-07-27 at 9 13 29 PM


I increased it a bit, let me know if its okay

Comment (anamatasanch) [reply src/main/webapp/css/style.css]: I just fixed it!

File: src/main/webapp/css/style.css (hunk 4 above)
Comment (anamatasanch) [reply src/main/webapp/css/style.css]: I just fixed it using the same code from the palette!

**Review by thuavu — APPROVED** [review]
LGTM!

**Comment by anamatasanch** [comment]
Made it so that dark colors have white text and light colors have black text.
darklight

**Review by anamatasanch — APPROVED** [review]

**Pull request merged**